ramls/records-editor.raml

```diff
@@ -0,0 +1,72 @@
+#%RAML 1.0
+
+title: QuickMARC Editor Records
+version: v1.0
+protocols: [ HTTP, HTTPS ]
+baseUri: http://github.com/folio-org/mod-quck-marc
+
+documentation:
+  - title: QuickMARC record API
+    content: This documents the API calls that can be made to work with MARC records
+
+types:
+  marcUi: !include schemas/marc_ui.json
+  errors: !include raml-storage/raml-util/schemas/errors.schema
+  UUID:
+    type: string
+    pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
+
+traits:
+  validate: !include raml-storage/raml-util/traits/validation.raml
+
+/records-editor/records:
+  get:
```

Comment:
Please, add `is[validate]` trait to validate format of id|instanceId.

Reply:
Done